code/modules/urist/items/miscitems.dm

```diff
@@ -574,3 +574,193 @@
 /obj/item/toy/torchmodel/urist
 	name = "table-top replica ship"
 	desc = "This is a replica of an unnamed exploration ship on a handsome wooden stand. Small lights blink on the hull and at the engine exhaust."
+
+//old hand radio
+/obj/item/device/radio/off/old
+	icon = 'icons/urist/structures&machinery/machinery.dmi'
+	icon_state = "portable_radio2"
+	item_state = "portable_radio2"
+
+//treasures meant as away loot, move some of these if they ever become a functional item like magic orbs/gems
+
+/obj/item/treasure
+	icon = 'icons/urist/items/misc.dmi'
+	throwforce = 0
+	throw_speed = 4
+	throw_range = 20
+	force = 0
+	w_class = 2
+
+/obj/item/treasure/tape
+	name = "retro holotape"
+	desc = "An old holotape cassette. Could be someone's mixtape or have valuable data."
+	icon_state = "tape1"
+
+/obj/item/treasure/tape/tape2
+	icon_state = "tape5"
+
+/obj/item/treasure/tape/tape3
+	icon_state = "tape6"
+
+/obj/item/treasure/coin
+	name = "\improper old copper coin"
+	desc = "A copper coin stamped with the image of a sailing ship."
+	icon_state = "oldcoin4"
+
+/obj/item/treasure/coin/platinum
+	name = "\improper old platinum coin"
+	desc = "A platinum coin stamped with the image of a king."
+	icon_state = "oldcoin1"
+
+/obj/item/treasure/coin/gold
+	name = "\improper old gold coin"
+	desc = "A gold coin stamped with the image of a castle."
+	icon_state = "oldcoin2"
+
+/obj/item/treasure/coin/silver
```

Comment:
there already are gold/silver/platinum/copper coins in the game, i don't really see the value in adding these versions. If you really want these alternate sprites, you could make them subtypes of /obj/item/material/coin/challenge so they don't change colour with assigned materials.